docs/bugsink/envelope-sammler.py

```
@@ -0,0 +1,64 @@
+        return self.rfile.read(int(self.headers.get("Content-Length") or 0))
+
+    def do_POST(self):
+        roh = self.rumpf_lesen()
+        if self.headers.get("Content-Encoding", "").lower() == "gzip":
+            try:
+                roh = gzip.decompress(roh)
+            except OSError:
+                pass
+        stempel = time.strftime("%Y%m%d-%H%M%S") + f"-{int(time.time()*1000)%1000:03d}"
+        pfad = os.path.join(ORDNER, f"envelope-{stempel}.txt")
+        with io.open(pfad, "wb") as f:
```

Ruff (0.16.3)

[warning] Use builtin `open`

Replace with builtin `open`

(UP020)